packages/dd-trace/src/config.js

```diff
@@ -539,6 +540,18 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
         process.env.DD_GIT_COMMIT_SHA,
         this.tags[GIT_COMMIT_SHA]
       )
+      if (!this.repositoryUrl || !this.commitSHA) {
```

Comment:
Not a blocker for this PR, but this is starting to be a lot of code in `config.js` which should probably be moved out at some point.